### PR TITLE
Add documentation for wiffi integration

### PR DESCRIPTION
**Pull request by mampfes**
**Description:**

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#30784

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/wiffi.markdown [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+title: Wiffi
+logo: wiffi.png
+description: Support for Wiffi devices from stall.biz, e.g. Weatherman, Rainyman, ...
```
Comment (frenck) [inline source/_integrations/wiffi.markdown]:
```suggestion
description: Support for WIFFI devices from stall.biz, e.g., Weatherman, Rainyman, ...
```

File: source/_integrations/wiffi.markdown [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+ha_category:
+  - DIY
+ha_release: 0.104
+ha_iot_class: Local Push
+ha_config_flow: true
+ha_codeowners:
+  - '@mampfes'
+---
+
+This integration allows you to connect your [Wiffi](https://stall.biz) devices directly to Home Assistant.
```
Comment (frenck) [inline source/_integrations/wiffi.markdown]:
```suggestion
This integration allows you to connect your [STALL WIFFI](https://stall.biz) devices directly to Home Assistant.
```

File: source/_integrations/wiffi.markdown [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+## Set up the integration via the integrations screen
+
+Menu: *Configuration* > *Integrations*
+
+Press on **wiffi** and configure the integration:
```
Comment (frenck) [inline source/_integrations/wiffi.markdown]:
```suggestion
Press on **WIFFI** and configure the integration:
```

File: source/_integrations/wiffi.markdown [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+* Enter the server port for json telegrams sent by the wiffi device.
```
Comment (frenck) [inline source/_integrations/wiffi.markdown]:
```suggestion
* Enter the server port for JSON telegrams sent by the WIFFI device.
```

File: source/_integrations/wiffi.markdown [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+## Configure wiffi device
```
Comment (frenck) [inline source/_integrations/wiffi.markdown]:
```suggestion
## Configure the WIFFI device
```

File: source/_integrations/wiffi.markdown [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+
+1. Set "CCU-IP Adresse myCCUIP" to IP address of Home Assistant.
```
Comment (frenck) [inline source/_integrations/wiffi.markdown]:
```suggestion
1. Set "CCU-IP Adresse myCCUIP" to the IP address of Home Assistant.
```

File: source/_integrations/wiffi.markdown [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+2. Set port for json telegrams to configured server port using parameter "send_json".
```
Comment (frenck) [inline source/_integrations/wiffi.markdown]:
```suggestion
2. Set port for JSON telegrams to configured server port using parameter "send_json".
```

File: source/_integrations/wiffi.markdown [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+
+Home Assistant will open a TCP server socket on the configured port and listens for incoming telegrams from wiffi devices. Entities from new devices will be automatically added.
```
Comment (frenck) [inline source/_integrations/wiffi.markdown]:
```suggestion
Home Assistant will open a TCP server socket on the configured port and listens for incoming telegrams from WIFFI devices. Entities from new devices will be automatically added.
```

File: source/_integrations/wiffi.markdown [hunk 9]
```diff
@@ -0,0 +1,57 @@
+---
+title: Wiffi
```
Comment (frenck) [inline source/_integrations/wiffi.markdown]:
```suggestion
title: STALL WIFFI
```

**Review by frenck — CHANGES_REQUESTED** [review]
Hi there @mampfes! 

I've reviewed your PR and left some comments, could you please take a look?

File: source/_integrations/wiffi.markdown (hunk 9 above)
Comment (mampfes) [reply source/_integrations/wiffi.markdown]: Done

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Review by frenck — DISMISSED** [review]

**Pull request merged**